dist/profile/manifests/openvpn.pp

```diff
@@ -60,13 +70,24 @@
     content => 'network: {config: disabled}',
   }
 
+  file { '/etc/netplan/':
+    ensure  => 'directory',
+    owner   => 'root',
+    group   => 'root',
+    recurse => true
+  }
+
   # Define eth interfaces with the correct mac addresses
   # We assume the parent folder already exists
-  file { '/etc/netplan/90-cloud-init.yaml':
+  file { '/etc/netplan/90-network-config.yaml':
     ensure => 'file',
     owner  => 'root',
     group  => 'root',
-    source => "puppet:///modules/${module_name}/openvpn/90-cloud-init.yaml",
+    content => template("${module_name}/openvpn/90-network-config.yaml.erb")
+  }
+
+    file { '/srv/demo/passwd':
```

Comment:
```suggestion
  file { '/srv/demo/passwd':
```

Reply:
What's this file '/srv/demo/passwd'?

Reply:
Copy paste to get a "template"  snippet, forgot to clean.
 I remove it.

Reply:
Done